connector-servicenow/src/main/java/com/redhat/cloud/notifications/connector/servicenow/ServiceNowRouteBuilder.java

```diff
@@ -48,8 +48,8 @@ public void configureRoute() {
 
     private void configureHttpsComponent() {
         HttpComponent httpComponent = getCamelContext().getComponent("https", HttpComponent.class);
-        httpComponent.setConnectTimeout(connectorConfig.getHttpsConnectTimeout());
-        httpComponent.setSocketTimeout(connectorConfig.getHttpsSocketTimeout());
+        httpComponent.setConnectTimeout(Timeout.ofMilliseconds(connectorConfig.getHttpsConnectTimeout()));
+        httpComponent.setSoTimeout(Timeout.ofMilliseconds(connectorConfig.getHttpsSocketTimeout()));
```

Comment:
`setSocketTimeout` has been renamed to `setSoTimeout` but that's the same method.